app/src/main/res/values/strings.xml

```diff
@@ -257,6 +257,7 @@
   <string name="admin_settings_cancel">Cancel</string>
   <string name="admin_settings_submit">Submit</string>
   <string name="admin_settings_incorrect">Incorrect Administrator PIN. Please try again.</string>
+  <string name="admin_settings_enter_user_new_pin">%1$s\'s New PIN.</string>
```

Comment:
Are we using it?

Reply:
yes here https://github.com/oppia/oppia-android/pull/2349#discussion_r550414387

Reply:
ok, just for the reference I am mentioning here Why we are introducing this. 

Earlier we use name directly from data binding but `TextInputLayout` is not supporting Data binding correctly so we are setting the string through kotlin file so we are introducing this new string item.

Reply:
I'm sorry i was litter confused here as the value for both string were the same 
I've changed it now